extension/css/cryptup.css

```diff
@@ -44,6 +44,7 @@ button.swal2-styled:focus,
   text-decoration: none;
```

Comment:
@limonte are CSS updates in this file looking good to you?

Reply:
yup, all good 👍